src/main/resources/alma/fix/otherFields.fix

```diff
@@ -159,3 +159,9 @@ do list(path: "542??", "var": "$i")
   copy_field("$i.u", "license[].$last.label")
   replace_all("license[].$last.label", "^(http://|https://)(.*)$","$2")
 end
+
+set_array("langNote[]")
+do list(path: "546??", "var": "$i")
+  copy_field("$i.a", "langNote[].$append")
+  copy_field("$i.b", "langNote[].$last")
```

Comment:
This would also overwrite a previous `546` in case the field is repeated, so I guess I'd have to veto this approach.

Reply:
This approach was not correct. I have to fix this.
But what do you mean by `546`

Reply:
> But what do you mean by `546`

MARC field `546`. If there are multiple occurrences, the last `546$$b` wins; previous ones will be overwritten.

Reply:
Thanks. Should be fixed now.